---
title: "Novembre Lab - Software"
layout: textlay
excerpt: "Novembre Lab at UChicago"
sitemap: false
permalink: /software.html
---

## Software

Many of our projects are available via the [Novembre Lab Github Page](https://github.com/NovembreLab). Here are some of our software projects:

* [Geography of Genetic Variants Browser](http://popgen.uchicago.edu/ggv/): A web-based tool for visualizing the geographic distribution of variants in several published datasets.

* [ADMIXTURE](http://www.genetics.ucla.edu/software/admixture/):  Software for ancestry inference developed by former UCLA PhD student David Alexander.

* [forqs](https://bitbucket.org/dkessner/forqs) (Forward-in-time simulation of Recombination, Quantitative traits, and Selection): A flexible forward-in-time simulator, developed by PhD student Darren Kessner.

* [harp](https://bitbucket.org/dkessner/harp) (Haplotype Analysis of Reads in Pools): Maximum likelihood estimation of frequencies of known haplotypes from pooled sequence data, developed by PhD student Darren Kessner.

* [RASPberry](https://bitbucket.org/phaentu/raspberry/) (Recombination via Ancestry Switch Probabilities): Software for recombination rate inference using ancestry switch points in admixed individuals, developed by former postdoc Daniel Wegmann.

* [ENCprime](https://github.com/jnovembre/ENCprime): A program that calculates a codon usage bias summary statistic, Nc’. It is based on the effective number of codons statistic Nc (or ENC) developed by Frank Wright, but improves upon it by accounting for background nucleotide composition.  

* [isoscatR](https://github.com/rundel/isoscatR): An R-package for spatial assignment using genetic and isotopic data, developed by former PhD student Colin Rundel, and based in part on the SCAT software from Matthew Stephens.

* [MDBlocks](http://ib.berkeley.edu/labs/slatkin/eriq/software/mdb_web/): A program written with Eric Anderson (when we were both at Berkeley) that partitions a chromosomal segment into haplotype blocks using biallelic, phased haplotype data.

* [SIMCOAL](http://cmpg.unibe.ch/software/simcoal/): A program that uses a coalescent framework to simulate the evolution of multiple haploid populations with complex demographies . Developed by Laurent Excoffier and Stefan Schnieder with some contributions from JN. NOTE: A new version, [SIMCOAL2](http://cmpg.unibe.ch/software/simcoal2/), is now available.


## Data Resources

* [Resources related to Novembre et al 2008](https://github.com/jnovembre/Novembre_etal_2008_misc):  Several files that aid in recreating the main figure from our paper:“Genes Mirror Geography in Europe” (Nature 956:98-101). Also the Geography of Genetic Variants Browser plots allele frequencies for the European subset of the data.  

* [African-American/African-Caribbean recombination maps](https://jnpopgen.org/software/software/AfricanAmerican_AfricanCaribbean_recombination_maps.zip): Recombination maps inferred from a sample of 2565 African Americans and 299 African Caribbeans.  From our paper: Wegmann et al (2011) Nature Genetics 43:847-53.

* Resources for Freedman et al 2014:  Fastq files are available via [SRA](http://www.ncbi.nlm.nih.gov/bioproject/PRJNA274504).  Auxiliary files such as vcf/bams are available via an sftp server. Please email – jnovembre at uchicago dot edu – for user and password.
