---
layout: post
title:  "Updated software and resources"
date:   2014-01-12
categories: news
---

Some updates of software/resources from the lab:

* I have just moved the ENCprime software over to github and updated <a href="https://github.com/jnovembre/ENCprime">[the link]() on the software/resources page.  This small piece of software computes a measure of codon usage bias for FASTA formatted sequences.

* I also added to the software/resources [a link]("https://github.com/jnovembre/Novembre_etal_2008_misc")> to a github repository with several files that help reproduce the main figure of the Novembre et al (2008) paper as from time to time I get requests for some help along those lines.

And in lab news, there have been some nice developments:

* Our team's paper on inferring migration origins of birds using genetics and isotopes [came out in Molecular Ecology]("http://onlinelibrary.wiley.com/resolve/openurl?genre=article&amp;sid=nlm:pubmed&amp;issn=0962-1083&amp;date=2013&amp;volume=22&amp;issue=16&amp;spage=4163") in the latter half of last year. Colin Rundel did excellent work on this project as part of his dissertation, and Kristen Ruegg and colleagues are doing awesome follow-up work on this project that will be exciting to share.  Molecular Ecology ran a [News and Views piece](http://onlinelibrary.wiley.com/doi/10.1111/mec.12441/full) on the work by Thor Veen.

* PhD student Eunjung Han's paper on assessing the impact of low coverage sequencing on population genetic inference has <a href="http://mbe.oxfordjournals.org/cgi/pmidlookup?view=long&amp;pmid=24288159">come out in Molecular Biology and Evolution</a>.

* PhD student Darren Kessner's software for forward simulation of selection on quantitative traits [forqs]("https://bitbucket.org/dkessner/forqs") is now described formally with [a software note published in Bioinformatics]("http://bioinformatics.oxfordjournals.org/content/early/2013/12/27/bioinformatics.btt712.long")

* The <a href="http://www.genetics.ucla.edu/software/admixture/">ADMIXTURE software</a> developed by David Alexander (now at Pac Bio) recently had its 200th citation.

* A collaborative paper led by Sebastian Zollner's group at University of Michigan, and recent PhD Matt Zawistowski in particular, has now been published in EJHG.  The paper is titled "<a href="http://www.ncbi.nlm.nih.gov/pubmed/24398795">Analysis of rare variant population structure in Europeans explains differential stratification of gene-based tests</a>" and it has very interesting results on how many of the novel gene-based tests perform in stratified mapping samples.
